src/snowflake/snowpark/dataframe.py

```diff
@@ -840,12 +836,12 @@ def __getitem__(self, item):
         elif isinstance(item, int):
             return self.__getitem__(self.columns[item])
         else:
-            raise TypeError(f"Unexpected item type: {type(item)}")
+            raise TypeError(f"Unexpected item type: {type(item)}")  # pragma: no cover
 
     def __getattr__(self, name):
         # Snowflake DB ignores cases when there is no quotes.
         if name.lower() not in [c.lower() for c in self.columns]:
-            raise AttributeError(
+            raise AttributeError(  # pragma: no cover
```

Comment:
Is this testable?

Reply:
Yea we should have negative test for this one